src/components/FormFields/index.js

```diff
@@ -90,6 +86,14 @@ class FormFields extends React.Component {
         this.setState({showNewEvents: !this.state.showNewEvents})
     }
 
+    setDefaultOrganization = () => {
+        const {user} = this.props;
+        const userType = get(user, 'userType')
+        const defaultOrganizationData = get(user, [`${userType}OrganizationData`, `${user.organization}`], {})
```

Comment:
We should use `_.get` sparingly. String path references to deeply nested objects are challenging to refactor without unexpected side effects.

Reply:
👍